src/PharmacyProduct.java

```diff
@@ -0,0 +1,27 @@
+public class PharmacyProduct extends Product {
+    private String newType;
+    private double refund;
+    private String latinName;
+
+    public PharmacyProduct(String name, double price, String newType, double refund, String latinName) {
+        super(name, price);
```

Comment:
you should use super constructor with tax and type (similarly as in FoodProduct)